README.md

```diff
@@ -11,6 +11,82 @@ This package is used to generate:
 
 **Warning!**  As the proposed ASHRAE Standard 205P has not yet been published, the content in this repository is subject to change and should be considered unstable for application development.
 
+Using DOIT Tasks!
+-----------------
+
+This repository ships with the [DoIt!](https://pydoit.org/) task automation tool.
+
+The following DoIt! tasks are available:
+
+- `doc`: Generates Markdown tables from common-scema
+- `render_template`: Demonstrate how to render a template using Jinja2 and the add_table hook.
+- `schema`: Generates JSON schema from common-scema
+- `test`: Performs unit tests and example file validation tests
+- `validate`: Validates common-schema against meta-schema
+
+Details of some of the tasks above are explained more below.
+
+### Rendering a Jinja Template: `render_template`
```

Comment:
I think the approach here is still in flux. It might be premature to document it all here. This README is getting really long, and I wonder if we should start using links to wiki pages instead.

Reply:
Sounds good. I can either remove it (and tuck it away somewhere) or leave it as is with the expectation that a future task will clean it up. Let me know your preference.